app/routes_helper.py

```diff
@@ -0,0 +1,19 @@
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
```

Comment:
Looks like we're not using any of these imports, can we please remove them so we don't expose our models. It could become difficult to maintain this in the future 😁